designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleStateStatus.scala

```diff
@@ -15,4 +19,104 @@ object SimpleStateStatus {
   val Failed: StateStatus = FailedStateStatus("FAILED")
   val Error: StateStatus = AllowDeployStateStatus("ERROR")
   val Warning: StateStatus = AllowDeployStateStatus("WARNING")
+
+  val defaultActions: List[ProcessActionType] = Nil
+
+  val actionStatusMap: Map[ProcessActionType, StateStatus] = Map(
+    ProcessActionType.Deploy -> SimpleStateStatus.Running,
+    ProcessActionType.Cancel -> SimpleStateStatus.Canceled,
+    ProcessActionType.Archive -> SimpleStateStatus.NotDeployed,
+    ProcessActionType.UnArchive -> SimpleStateStatus.NotDeployed
+  )
+
+  val statusActionsMap: Map[StateStatus, List[ProcessActionType]] = Map(
+    SimpleStateStatus.Unknown -> List(ProcessActionType.Deploy),
+    SimpleStateStatus.NotDeployed -> List(ProcessActionType.Deploy, ProcessActionType.Archive),
+    SimpleStateStatus.DuringDeploy -> List(ProcessActionType.Deploy, ProcessActionType.Cancel),
+    SimpleStateStatus.Running -> List(ProcessActionType.Cancel, ProcessActionType.Pause, ProcessActionType.Deploy),
+    SimpleStateStatus.Canceled -> List(ProcessActionType.Deploy, ProcessActionType.Archive),
+    SimpleStateStatus.Restarting -> List(ProcessActionType.Deploy, ProcessActionType.Cancel),
+    SimpleStateStatus.Finished -> List(ProcessActionType.Deploy, ProcessActionType.Archive),
+    // When Failed - process is in terminal state in Flink and it doesn't require any cleanup in Flink, but in NK it does
+    // - that's why Cancel action is available
+    SimpleStateStatus.Failed -> List(ProcessActionType.Deploy, ProcessActionType.Cancel),
+    SimpleStateStatus.Error -> List(ProcessActionType.Deploy, ProcessActionType.Cancel),
+    SimpleStateStatus.Warning -> List(ProcessActionType.Deploy, ProcessActionType.Cancel),
+    SimpleStateStatus.FailedToGet -> List(ProcessActionType.Deploy, ProcessActionType.Archive)
```

Comment:
Isn't it just SimpleStateStatus.Unknown?

Reply:
Yes, for me also it looks like Unknown. In this change I want to keep current state definitions, in the next step I want to simplify them (due to this state redundancy). To avoid complications it should happen after https://github.com/TouK/nussknacker/pull/4104